src/crypto/PSASessionKeystore.cpp

```diff
@@ -24,62 +24,54 @@ namespace Crypto {
 
 namespace {
 
-class AesKeyAttributes
+class keyAttributesBase
```

Comment:
This is a minor, but why does this class name start with lower case?

Reply:
must have make it lowercase by mistake. fixed